modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/mapper/PageFragmentInstanceDefinitionMapper.java

```diff
@@ -686,8 +672,23 @@ private FragmentImageClassPKReference _toFragmentImageClassPKReference(
 		};
 	}
 
+	private FragmentInlineValue _toFragmentInlineValue(
+		boolean acceptAllLanguages, JSONObject jsonObject, Locale locale) {
+
+		return new FragmentInlineValue() {
+			{
+				value = jsonObject.getString(locale.toString());
+
+				if (acceptAllLanguages) {
+					setValue_i18n(_toLocalizedValues(jsonObject));
```

Comment:
why set instead of assignment?

Reply:
Totally right. I had this pattern in mind for larger entities where there was a lot of fields to assign and we can not assign them alphabetically without breaking the flow with the if.

I'll change it in a moment